articles/postgresql/flexible-server/how-to-restore-dropped-server.md

```diff
@@ -40,6 +40,9 @@ To restore a dropped Azure Database for PostgreSQL Flexible server, you need
 
 4. Browse to the PostgreSQL [Create Server REST API Page](/rest/api/postgresql/flexibleserver/servers/create) and select the **Try It** tab highlighted in green. Sign in with your Azure account.
 
+ > [!Important]
+    > Use this api-version **_2023-03-01-preview_** rather than the default before running to enable this API function as expected as detailed in the following step.
```

Comment:
```suggestion
   > [!Important]
   > Use this api-version **_2023-03-01-preview_** rather than the default before running to enable this API function as expected as detailed in the following step.
```